credentials/apps/catalog/management/commands/copy_catalog.py

```diff
@@ -41,10 +42,28 @@ def handle(self, *args, **options):
 
             logger.info('Copying catalog data for site {}'.format(site.domain))
             client = site_config.catalog_api_client
-            Command.fetch_programs(site, client, page_size=page_size)
-            logger.info('Finished copying programs.')
-            Command.fetch_pathways(site, client, page_size=page_size)
-            logger.info('Finished copying pathways.')
+            try:
+                Command.fetch_programs(site, client, page_size=page_size)
+                logger.info('Finished copying programs.')
+            except Exception as error:
+                success = False
+                logger.info('Failed to copy programs for site: {} with error: {}'.format(
+                    site.domain,
+                    error
+                ))
+
+            try:
+                Command.fetch_pathways(site, client, page_size=page_size)
+                logger.info('Finished copying pathways.')
+            except Exception as error:
+                success = False
+                logger.info('Failed to copy pathways for site: {} with error: {}'.format(
+                    site.domain,
+                    error
+                ))
+
+        if not success:
+            raise CommandError('One or more of the sites loaders above failed.')
```

Comment:
`Programs / Pathways copying for 3 out of 6 sites have failed. Please update the configuration.`

Reply:
Done